docs/USAGE.md

```diff
@@ -423,4 +425,39 @@ in the new format like so,
     def apply(self, block_value):
         for child_block in formatted_list_child_generator(list_block_value):
             ...
-```
\ No newline at end of file
+```
+
+# Using Management Commands
+
+## streamdatamigration
+
+A management command to generate data migrations for a given operation and multiple block paths.
+(Currently limited to a single app at a time.) For help,
+
+```
+python manage.py streamdatamigration --help
+```
+
+or help for specific operations,
+
+```
+python manage.py streamdatamigration <operation> --help
+```
+Currently this only supports rename and remove operations.
+
+A rename operation for a block used in multiple streamfields in an app would look like this,
+
+```
+python manage.py streamdatamigration rename <app_label> <old_name> <new_name> <path1> <path2> <path3> ...
```

Comment:
Is this missing the model and field names?

Reply:
I think that's included in the `path` in this case (`where `<path>` would look like `<model_name>.<streamfield_name>....<parent_block_name>``)

Reply:
Yep, I see you are correct :+1:

Reply:
they're all part of the <path>, I mentioned what the path looks like below cuz otherwise it's very long for a line :sweat_smile: 
would it be better to put everything in one line?